src/website/website.js

```diff
@@ -79,3 +81,39 @@ function readThemeSettings (filepath) {
     throw new Error('Failed to parse JSON theme settings file: ' + filepath)
   }
 }
+
+function outputSEO (output, seoLocation, rootAlbum) {
+  // SEO (sitemap and robots.txt)
+  if (!seoLocation) {
+    return
+  }
+
+  // Guaranteed to end with slash
+  const seoPrefix = seoLocation + (seoLocation.endsWith('/') ? '' : '/')
+
+  // Generate robots.txt
+  const robotsTxt = 'User-Agent: *\nDisallow:\n\nSitemap: ' + seoPrefix + 'sitemap.xml\n'
+  fs.writeFile(path.join(output, 'robots.txt'), robotsTxt, () => {
+  })
+
+  // Generate sitemap
+  let sitemapXML = '<?xml version="1.0" encoding="UTF-8"?>\n' +
+    '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">\n'
+  const now = new Date().toISOString()
+
+  function addToSitemap (album) {
+    sitemapXML +=
+      '    <url>\n' +
+      '        <loc>' + seoPrefix + album.url + '</loc>\n' +
+      '        <lastmod>' + now + '</lastmod>\n' +
```

Comment:
I wonder if there's a smart way to calculate the real "last mod" time for each album. I don't think it's the latest photo date, e.g. when we add an old photo to a recent album.

This is something we could improve later (if feasible, I'm not sure).

Reply:
My thought process exactly :)